ocs_ci/ocs/ocp.py

```diff
@@ -106,6 +106,34 @@ def exec_oc_cmd(self, command, out_yaml_format=True, secrets=None, **kwargs):
             return yaml.safe_load(out)
         return out
 
+    def exec_oc_debug_cmd(self, node, cmd_list):
+        """
+        Function to execute "oc debug" command on OCP node
+
+        Args:
+            node (str): Node name where the command to be executed
+            cmd_list (list): List of commands eg: ['cmd1', 'cmd2']
+
+        Returns:
+            out (str): Returns output of the executed command/commands
+
+        Raises:
+            CommandFailed: When failure in command execution
+        """
+        # Appending one empty value in list for string manipulation
+        cmd_list.append(' ')
+        cmd = " || echo 'CMD FAILED';".join(cmd_list)
+        debug_cmd = f"debug nodes/{node} -- chroot /host /bin/bash -c \"{cmd}\""
+        out = str(self.exec_oc_cmd(
+            command=debug_cmd, out_yaml_format=False
+        ))
+        if out.__contains__('CMD FAILED'):
+            logging.error(f"{debug_cmd} \n {out}")
+            raise CommandFailed
+        else:
+            logging.info(f"{debug_cmd} \n {out}")
+            return out
```

Comment:
In the output we will have: 
```
Starting pod/ip-10-0-132-136us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`
```
Do you think that it will be worth to remove this from the out before returning?

Reply:
`out` will not have the first and last 2 lines. It has only the command output result, please refer success/failed output local run logs in earlier comments.
```Starting pod/ip-10-0-132-136us-east-2computeinternal-debug ...
                                                         To use host binaries, run `chroot /host`
                                                         
                                                         Removing debug pod ...
```

Reply:
Ok, so looks like this output is printed out to stderr. Cool.